specification/resources/resource-manager/Microsoft.Resources/stable/2022-01-01/subscriptions.json

```diff
@@ -0,0 +1,739 @@
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns an array of subscriptions.",
+            "schema": {
+              "$ref": "#/definitions/SubscriptionListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "GetAllSubscriptions": {
+            "$ref": "./examples/GetSubscriptions.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/tenants": {
+      "get": {
+        "tags": [
+          "Tenants"
+        ],
+        "operationId": "Tenants_List",
+        "description": "Gets the tenants for your account.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns an array of tenants.",
+            "schema": {
+              "$ref": "#/definitions/TenantListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "GetAllTenants": {
+            "$ref": "./examples/GetTenants.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/providers/Microsoft.Resources/checkResourceName": {
+      "post": {
+        "tags": [
+          "Tenants"
+        ],
+        "operationId": "checkResourceName",
+        "description": "A resource name is valid if it is not a reserved word, does not contains a reserved word and does not start with a reserved word",
+        "summary": "Checks resource name validity",
+        "consumes": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/CheckResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns status as allowed or not.",
+            "schema": {
+              "$ref": "#/definitions/CheckResourceNameResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "CheckValidityForAResourceName": {
+            "$ref": "./examples/CheckResourceName.json"
```

Comment:
LintDiff is complaining about this CheckResourceName.json example not existing. Does this need to be created still? Also keep in mind the other x-ms-examples .json files must be present.